message_definitions/v1.0/ardupilotmega.xml

```diff
@@ -113,7 +113,7 @@
       </entry>
       <entry value="42424" name="MAV_CMD_DO_START_MAG_CAL">
         <description>Initiate a magnetometer calibration.</description>
-        <param index="1" label="bitmask" increment="1">uint8_t bitmask of magnetometers (0 means all).</param>
+        <param index="1" label="bitmask" minValue="0" maxValue="255" increment="1">Bitmask of magnetometers to calibrate (0 means all).</param>
```

Comment:
Looks good. Is it helpful for anyone to understand that this maps to a `uint8_t` (ie did removing that hurt anyone). I don't think so, in particular because uint8_t is a C type not a MAVLink type) but this is a sanity check.